styles.css

```diff
@@ -468,15 +448,13 @@ Alerts  */
   color: black;
   font-size: 80%;
   font-weight: 700;
-  /* padding: 5px 10px; */
 }
 
 .card-dismiss-btn:hover {
   padding: 10px 15px;
```

Comment:
Use `rem` even for padding. Makes them more consistent.

Reply:
Yes, will avoid using px as suggested by Kushagra too.